client/js/components/statement/publicStatementLists/DpPublicStatement.vue

```diff
@@ -23,15 +23,18 @@
               :id="number"
               name="item_check[]"
               :value="id">
-            <label
-              :for="number"
-              data-cy="statementNumber"
-              class="inline u-mb-0 u-ml-0_25">{{ number || '' }}</label>
+            <span
+              class="c-public-statement__tooltip"
+              v-tooltip="renderTooltipContent(tooltipContent)">
+              <label
+                :for="number"
+                data-cy="statementNumber"
+                class="inline u-mb-0">
+                {{ `#${ number || externId }` }}
```

Comment:
I'd suggest to remove the `#` hash character here. We could as well use an info icon as the tooltip trigger (positioned after the ID).

Another question: isn't "number" and "externId" the same value?

Reply:
you have a suggestion, which icon i could use here? I would suggest just to remove the hash.

No, they are different, but its difficult to explain it here

Reply:
I decided to remove it. done here https://github.com/demos-europe/demosplan-core/pull/2832/commits/da48392fa44c9bf01263c06810ee84d435a39fc3